### PR TITLE
Add theme designer to v9 docsite

### PR DESCRIPTION
**Pull request by micahgodbolt**
David (and others) were asking about making the theme designer accessible via the public docsite. This PR creates a customized story to embed the theme designer, also allowing an aka link to point directly at the created iframe.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9e7682f713135742648c4a6144bce5135048d990'>9e7682f713135742648c4a6144bce5135048d990</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by tudorpopams** [comment]
Hi @micahgodbolt , can you give us more context about this PR?

File: apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx [hunk 1]
```diff
@@ -14,6 +14,9 @@ interface IFluentDocsContainerProps {
 export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
   // eslint-disable-next-line deprecation/deprecation
   const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
+  if (context.id === 'theme-theme-designer--page') {
```
Comment (Hotell) [inline apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx:17]: this code looks brittle, what is the logic behind this ? can it be implemented in some better way? can we provide docs ?

File: apps/public-docsite-v9/src/Theme/ThemeDesigner.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#docs-root .sbdocs-content {
```
Comment (Hotell) [inline apps/public-docsite-v9/src/Theme/ThemeDesigner.css:1]: why is this needed ? "overriding" storybook magic globals is not very good idea as things might break (especially if we dont have any test coverage )

Comment (micahgodbolt) [reply apps/public-docsite-v9/src/Theme/ThemeDesigner.css:1]: My goal is to hide the theme selector and allow the story to be full width....otherwise you get this:

image

I'll work on a less hacky way to do it.

Comment (micahgodbolt) [reply apps/public-docsite-v9/src/Theme/ThemeDesigner.css]: Ok, I moved both the theme removal and the full screen functionality to reusable parameters.

**Review by tudorpopams — APPROVED** [review]

File: apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx [hunk 3]
```diff
@@ -14,19 +14,27 @@ interface IFluentDocsContainerProps {
...
+
   return (
-    <>
-      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={selectedTheme?.theme ?? webLightTheme}>
-        <FluentDocsHeader
-          // eslint-disable-next-line deprecation/deprecation
-          storybookGlobals={context.globals}
-        />
-      </FluentProvider>
+    <div className={context.parameters.fullScreen && 'full-screen'}>
```
Comment (Hotell) [inline apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx:21]: this component has lot of unrelated logic. we should extract that to single source of truth. something like following:

```tsx
export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
  const content =
    context.parameters.hideThemeSelector === true ? (
      <DocsContainer context={context}>{children}</DocsContainer>
    ) : (
      <EnhancedDocsContainer context={context}>{children}</EnhancedDocsContainer>
    );

  return <div className={context.parameters.fullScreen && 'full-screen'}>{content}</div>;
};

const EnhancedDocsContainer: React.FC<IFluentDocsContainerProps> = props => {
  const { context, children } = props;
  const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
  return (
    <>
      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={selectedTheme?.theme ?? webLightTheme}>
        <FluentDocsHeader storybookGlobals={context.globals} />
      </FluentProvider>

      <FluentProvider style={{ backgroundColor: 'transparent' }} theme={webLightTheme}>
        <DocsContainer context={context}>{children}</DocsContainer>
      </FluentProvider>
    </>
  );
};

```

File: apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx [hunk 4]
```diff
@@ -14,19 +14,27 @@ interface IFluentDocsContainerProps {
 export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
   // eslint-disable-next-line deprecation/deprecation
   const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
+
```
Comment (Hotell) [inline apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx:17]: question(unrelated to this PR): any idea why are we naming files with `stories` suffix that are no stories at all ?

**Comment by Hotell** [comment]
what is the funcionality of red border highlighted items ?  

image

I'm confused about those, wondering what customers will think. can we improve this ?

**Comment by micahgodbolt** [comment]
TBH, i agree that the theme designer isn't ready to be published on the main docsite. I'll close this PR down and create an aka link instead for those that need to see the latest theme-designer.

**Comment by micahgodbolt** [comment]
https://aka.ms/themedesigner-v9
